esp/templates/program/receipts/default.html

```diff
@@ -58,9 +58,9 @@ <h1>Registration Receipt for {{ user.first_name }} {{ user.last_name }}</h1>
 Program Information: <br />
 <ul>
 <li>Name: {{ program.niceName }}</li>
-<li>Date Range: {{ program.date_range }}</li>
+<li>Date{% if program.datetime_range.1 %} Range{% endif %}: {{ program.datetime_range.0|date:"D M j Y" }}{% if program.datetime_range.1 %} - {{ program.datetime_range.1|date:"D M j Y" }}{% endif %}</li>
```

Comment:
I'm still seeing a range even though I have a one-day program. Is it possible this is checking for multiple times instead of just for multiple days? 

![image](https://github.com/learning-unlimited/ESP-Website/assets/25392075/eed08e29-b035-471b-811e-bcb9709f7f19)